device-types/Dell/PowerEdge_R740.yaml

```diff
@@ -34,6 +41,15 @@ interfaces:
     type: 1000base-t
   - name: Gig-E 4
     type: 1000base-t
-  - name: iDRAC
+  - name: iDRAC9
     type: 1000base-t
     mgmt_only: true
+device-bays:
+  - name: PCIe-Gen3 1
+  - name: PCIe-Gen3 2
+  - name: PCIe-Gen3 3
+  - name: Riser2 PCIe-Gen3 4
+  - name: Riser2 PCIe-Gen3 5
+  - name: Riser2 PCIe-Gen3 6
+  - name: Riser3 PCIe-Gen3 7
+  - name: Riser3 PCIe-Gen3 8
```

Comment:
Would be more appropriate as module bays, however watch the [discussion](https://github.com/netbox-community/netbox/discussions/9238) for a final verdict.

Reply:
You have over 100 PCIe cards available from NIC, HBA, GPU, etc

Reply:
You don't need to create each NIC/HBA/GPU yourself, but Modules allow more flexibility for most users.  The final decision on this is going to be use module bays.